crypto/hpke/hpke.c

```diff
@@ -599,9 +600,18 @@ static int hpke_get_grease_value(OSSL_LIB_CTX *libctx, const char *propq,
         ERR_raise(ERR_LIB_CRYPTO, ERR_R_INTERNAL_ERROR);
         goto err;
     }
+    aead_info = ossl_HPKE_AEAD_INFO_find_id(chosen.aead_id);
+    if (aead_info == NULL) {
+        erv = 0;
+        ERR_raise(ERR_LIB_CRYPTO, ERR_R_INTERNAL_ERROR);
+        goto err;
+    }
     if ((crv = hpke_suite_check(chosen)) != 1)
         return 0;
```

Comment:
I would suggest being consistent with what you do here with returns.. i.e you use goto above in this function.. Generally I use return until something needs to be freed.. and then use goto after that

Reply:
fair point - that pattern is my goal too, I'm just imperfect:-)

Reply:
fixed